src/routes/ConsortiumManager/settings/authorization-roles/UsersCapabilities/UsersCapabilitiesCompareItems/UsersCapabilitiesCompareItems.test.js

```diff
@@ -0,0 +1,122 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import {
+  render,
+  screen,
+} from '@folio/jest-config-stripes/testing-library/react';
+import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
+import {
+  useAuthorizationRoles,
+  useUserCapabilities,
+  useUserCapabilitiesSets,
+  useUserRolesByUserIds
+} from '@folio/stripes-authorization-components';
+
+import {
+  tenants,
+  groupedRoleCapabilitiesByType,
+  groupedRoleCapabilitySetsByType,
+} from 'fixtures';
+import { ConsortiumManagerContextProviderMock } from 'helpers';
+
+import { COMPARE_ITEM_NAME } from '../../../users/general/PermissionSets/PermissionSetsCompare/constants';
+import { UsersCapabilitiesCompareItems } from './UsersCapabilitiesCompareItems';
+import  { useUsers } from '../../../../../../hooks';
```

Comment:
this should go line 22

Reply:
Why it should go on line 22 ? If we follow alphabetical order

Reply:
alphabetical order + hierarchical. Since the file goes to top level a lot